homeassistant/components/sonos/number.py

```diff
@@ -34,7 +35,9 @@ async def async_setup_entry(
 ) -> None:
     """Set up the Sonos number platform from a config entry."""
 
-    def available_soco_attributes(speaker: SonosSpeaker) -> list[str]:
+    def available_soco_attributes(
+        speaker: SonosSpeaker,
+    ) -> list[tuple[str, tuple[int, int]]]:
```

Comment:
Maybe create an alias like `SocoFeatures = list[tuple[str, tuple[int, int]]]` for readability?